models/containerized_model/DEPLOYMENT.md

```diff
@@ -0,0 +1,90 @@
+# How to deploy the "containerized" model to Kubernetes?
+
+## Pre-requisites
+- A Containerized model.
+- `kubectl`.
+- Private docker registry to push containerized image to.
+- A valid `kubeconfig` file with the current context activated.
+
+## Step 1 - Push image to registry (private)
+The containerized image that has been built will need to be pushed to a private docker registry in order for the `k8s` cluster to pull it from.
+
+### Pre-requisites
+- Private docker registry to push containerized image to. This should be the registry that your`k8s` cluster is configured with.
+- Make sure you have write-access to the registry and are authenticated.
+
+### Tag docker image
+```
+docker tag <docker-image> <your-private-registry-url>:<docker-image-with-tag>
+```
+
+### Push the image
+```
+docker push <your-private-registry-url>:<docker-image-with-tag>
+```
+
+## Step 2 - Update deployment configuration and credentials
+Update `MODEL_PATH`,  cloud storage credentials (e.g s3) and the `docker image` name in the `generated-container-model/deployment.yml` file.
+Below is the snippet from `generated-container-model/deployment.yml` that needs to be updated:
+```
+spec:
+  containers:
+    - image: "<your-docker-image>"
+      imagePullPolicy: IfNotPresent
+      name: {{DEPLOYMENT_NAME}}
+      ports:
+        - containerPort: 8551
+          protocol: TCP
+      env:
+        - name: MODEL_PATH
```

Comment:
deployment file has placeholdrs for only S3 and not for azure blob or gcs bucket, is it expected?

Reply:
I think we've kept it since the customer uses Minio. I'll let @cdraper-CS comment if we need to add other stuff as well.